### PR TITLE
[feat] BulkSender (Airdrop)

### PR DESCRIPTION
**Pull request by roynalnaruto**
https://gist.github.com/roynalnaruto/a797933084ff96a65adc8e1bac6e3f48

### CODE REVIEW

**Comment by roynalnaruto** [comment]
@tymat Please refer to the gist and the code, let me know if I can change the strategy

**Comment by mrenoon** [comment]
We have no need for the TokenSwap contract yet. Best to just add the BulkSender now, leave the TokenSwap in a branch that we might revisit later.
This TokenSwap contract doesn't apply for our context (since the tokens must always be minted from the PoP process)

File: contracts/Airdrop/BulkSender.sol [hunk 1]
```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.4.19;
+
+import "@digix/core2/contracts/common/ERCTwenty.sol";
```
Comment (mrenoon) [inline contracts/Airdrop/BulkSender.sol]: lets use a ERCTwenty interface that is not from our core2. You can just use the open-zeppelin one or just copy paste from core2. This is like a separate, open source contract tool that anyone could use, with no link to our system.

**Comment by roynalnaruto** [comment]
@mrenoon this branch only has the `BulkSender` contracts and tests

**Pull request merged**